gui/rendering/ShaderProgram.cpp

```diff
@@ -7,70 +7,66 @@
 #include "Utils.hpp"
 
 rendering::ShaderProgram::ShaderProgram(const char *vertexProgram,
```

Comment:
since you convert these const char* to std::string anyway, can't we remove this constructor and just use the std::string one below?

Reply:
yes, it's legacy code from the previous implementation.